### PR TITLE
new raw issuance rpc

### PR DESCRIPTION
**Pull request by tomt1664**
A new RPC that generates an unsigned raw issuance transaction based on supplied addresses and amounts, and the input outpoint containing policyAsset to spend from (this is needed to generate the asset and token IDs). 

The raw transaction can then be signed directly with signrawtransaction.

### CODE REVIEW

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -3764,6 +3764,113 @@ UniValue claimpegin(const JSONRPCRequest& request)
...
+  bool fBlindIssuances = false;
+
+  //get the input outpoint from RPC
+  uint256 prevtxhash;
+  prevtxhash.SetHex(request.params[7].get_str());
+  uint32_t nout = atoi(request.params[8].get_str());
+  COutPoint policyOutpoint(prevtxhash,nout);
+
+  uint256 dummyentropy;
+  CAsset dummyasset;
```
Comment (nkostoulas) [inline src/wallet/rpcwallet.cpp]: These don't seem to be used

**Comment by nkostoulas** [comment]
Would we need something similar for re-issuance once the unblinded re-issuance bug is resolved?

**Comment by nkostoulas** [comment]
[Additional Ocean features](https://trello.com/c/0vZEwbFO/76-additional-ocean-features)

**Comment by tomt1664** [comment]
I have added some pull tests for multisig tx generation and issuance in wallet.py
The RPC now has an option for creating multiple polcyasset change outputs.

**Pull request merged**